src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Resolvers/DynamicObjectResolver.cs

```diff
@@ -940,17 +958,22 @@ private static void BuildDeserialize(Type type, ObjectSerializationInfo info, IL
                     .Select(item => new DeserializeInfo
                     {
                         MemberInfo = item,
-                        LocalField = il.DeclareLocal(item.Type),
+                        LocalField = canOverwrite ? default : il.DeclareLocal(item.Type),
+                        LocalIsInitializedField = canOverwrite ? default : il.DeclareLocal(typeof(bool)),
```

Comment:
This sounds like it identifies a *field* that indicates whether a local is initialized. But it's a local itself -- not a field.
I guess the pre-existing name of `LocalField` is similarly misnamed. But perhaps now is a good time to fix them both.

Reply:
LocalField -> LocalVariable
LocalIsInitializedField -> IsInitializedLocalVariable

Fixed.